web-ui/view/documentation.ml

```diff
@@ -103,3 +106,177 @@ let getting_started =
             ];
         ];
     ]
+
+let user_guide =
+  Template_v1.instance
+    [
+      div
+        ~a:[ a_class [ "grid grid-cols-6" ] ]
+        [
+          div
+            ~a:[ a_class [ "blog-sidebar" ] ]
+            [
+              div
+                ~a:[ Tyxml_helpers.x_data "{submenu: false}" ]
+                [ (* this is where we would have the sidebar links if we had any *) ];
+            ];
+          div
+            ~a:[ a_class [ "prose col-span-4" ] ]
+            [
+              h1 [ txt "Documentation" ];
+              p
+                [
+                  txt
+                    "This is a Work In Progress. Thank you for your patience \
+                     as we update it with detailed documentation covering each \
+                     of the pages of ocaml-ci.";
```

Comment:
Should we be clear that the `ocaml-ci` project globally is also a WIP, not just the documentation?

Reply:
This appears in the Getting Started page so I think we can skip it here.